### DISCUSSION TITLE
Prevent auto install of mods for those who have limited RAM.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3966] [Ideas]
So basically, I'm talking about curated maps that the game makes you install when they first come out. Seen a few people who only have 4 GB of RAM have loading problems because they have auto-installed curated maps built up from over the past few months/years. No idea if it is possible to check the amount of RAM available on a user's computer, but if it is possible, then there will be less support posts and frustration.

**Comment by DrDallas1** [comment]
They cant manually delete them from the game files and unsubscribe from them? I havent had to download any curated map... (except when they just come out).

**Reply by Jdance-Media** [reply]
True, looking back, my idea isn't the best, but it still requires them to make a support post. Since you can't expect newer players or those with a low-end computer to know every detail about how the workshop works.
